run.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+cd /usr/src/app
+case ${DOCKER_STATE} in
+migrate)
+    echo "running migrate"
+    bundle exec rake db:migrate
+    ;;
+seed)
+    echo "running seed"
+    bundle exec rake db:migrate
+    bundle exec rake db:seed
+    ;;
+vagrant)
+    echo "running seed"
+    bundle exec rake db:create
+    bundle exec rake db:seed
+    bundle exec rake db:migrate
+    ;;
+create)
+    echo "running seed"
+    bundle exec rake db:create
+    bundle exec rake db:migrate
+    bundle exec rake db:seed
+    ;;
+esac
+bundle exec rails s
```

Comment:
As discussed, let's ditch runit and switch this line to:
`bundle exec unicorn -p $UNICORN_PORT -c ./config/unicorn.rb`

Also check that the config/unicorn.rb exists in the app and at least sets the number of workers.